docs/components/nav_drawer.md

```diff
@@ -0,0 +1,324 @@
+<!--docs:
+title: "Material <component>"
```

Comment:
This should be `Material Navigation Drawer` or something like that

Reply:
Done